test/EVMHost.h

```diff
@@ -49,8 +49,9 @@ class EVMHost: public evmc::MockedHost
 	/// Tries to load all defined evmc vm shared libraries.
 	/// @param _vmPaths paths to multiple evmc shared libraries.
 	/// @throw Exception if multiple evm1 or multiple ewasm evmc vms where loaded.
-	/// @returns true, if an evmc vm was supporting evm1 loaded properly.
-	static bool checkVmPaths(std::vector<boost::filesystem::path> const& _vmPaths);
+	/// @returns A pair of booleans, the first element being true, if an evmc vm supporting evm1 was loaded properly,
+	///          the second being true, if an evmc vm supporting ewasm was loaded properly.
+	static std::pair<bool, bool> checkVmPaths(std::vector<boost::filesystem::path> const& _vmPaths);
```

Comment:
Why not `tuple`? Not that it matters all that much, but I think it's now recommended over `pair`.
```suggestion
	static std::tuple<bool, bool> checkVmPaths(std::vector<boost::filesystem::path> const& _vmPaths);
```

Reply:
*Is* that a general recommendation? There is some merit to it, especially if one doesn't want to use ``first`` and ``second`` anyways like here. Doesn't matter too much anyways, I'll change it.

Reply:
Good question. I thought it was and that I've seen it somewhere in [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md) but apparently not. [It's mentioned in our CODING_STYLE](https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md) but it says that `pair` is fine too:

> If a function returns multiple values, use std::tuple (std::pair acceptable) or better introduce a struct type. Do not use */& arguments.

So yeah, you have a point. I generally thought of `pair` as legacy stuff but looks like it was just my subjective impression rather than a general recommendation.